src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -1113,3 +1171,98 @@ def plot_scatterplot(
         buffer.seek(0)
         self._data = self._data.reset_index()
         return Image.from_bytes(buffer.read())
+
+    def split_rows(self, percentage_in_first: float) -> tuple[TimeSeries, TimeSeries]:
+        """
+        Split the table into two new tables.
+
+        The original table is not modified.
+
+        Parameters
+        ----------
+        percentage_in_first : float
+            The desired size of the first table in percentage to the given table; must be between 0 and 1.
+
+        Returns
+        -------
+        result : (Table, Table)
```

Comment:
```suggestion
        result : (TimeSeries, TimeSeries)
```

Reply:
```suggestion
        result:
```
[We do not want to add type hints in the docstrings anymore](https://library.safeds.com/en/latest/development/project_guidelines/#docstrings)